docs/tutorials/go-built-in.md

```diff
@@ -380,14 +380,13 @@ func main() {
  }
 
  node.Start()
- defer func() {
-  node.Stop()
-  node.Wait()
- }()
 
  c := make(chan os.Signal, 1)
  signal.Notify(c, os.Interrupt, syscall.SIGTERM)
  <-c
+
+ node.Stop()
+ node.Wait()
  os.Exit(0)
```

Comment:
Since this is `main` anyway, and exiting with a zero status is the default, might we not do this more simply by deleting the `os.Exit(0)` call?